packages/sitecore-jss-react/src/index.ts

```diff
@@ -45,6 +45,7 @@ export {
 } from '@sitecore-jss/sitecore-jss/i18n';
 export { mediaApi } from '@sitecore-jss/sitecore-jss/media';
 export { getFEAASLibraryStylesheetLinks } from '@sitecore-jss/sitecore-jss/feaas';
+export { getComponentLibraryStylesheetLinks } from '@sitecore-jss/sitecore-jss/layout';
```

Comment:
Should this statement be added under "@sitecore-jss/sitecore-jss/layout" that is used above?